pypreprocess/subject_data.py

```diff
@@ -141,6 +142,10 @@ def __init__(self, func=None, anat=None, subject_id="sub001",
         self.nipype_results = {}
         self._set_items(**kwargs)
         self.scratch = output_dir if scratch is None else scratch
+        self.anat_scratch_dir = anat_output_dir if scratch is None else scratch
+        self.session_scratch_dirs = \
```

Comment:
Avoid using "\". You can break before an if or and else, and then you don't need the abnoxious "\".